test/l1ShareBridge/L1SharedBridgeHyperEnabled.t.sol

```diff
@@ -0,0 +1,209 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import {L1SharedBridgeTest} from "./_L1SharedBridge_Shared.t.sol";
+
+import {ETH_TOKEN_ADDRESS} from "@era-contracts/l1-contracts/contracts/common/Config.sol";
+import {IBridgehub} from "@era-contracts/l1-contracts/contracts/bridgehub/IBridgehub.sol";
+import {L2Message, TxStatus} from "@era-contracts/l1-contracts/contracts/common/Messaging.sol";
+import {IMailbox} from "@era-contracts/l1-contracts/contracts/state-transition/chain-interfaces/IMailbox.sol";
+import {IL1ERC20Bridge} from "@era-contracts/l1-contracts/contracts/bridge/interfaces/IL1ERC20Bridge.sol";
+import {L2_BASE_TOKEN_SYSTEM_CONTRACT_ADDR} from "@era-contracts/l1-contracts/contracts/common/L2ContractAddresses.sol";
+
+// note, this should be the same as where hyper is disabled
+contract L1SharedBridgeHyperEnabledTest is L1SharedBridgeTest {
```

Comment:
This HyperEnabled was a mistake even in our repo, it was the same as the Base case, does it add to coverage or test anything in a different way?

Reply:
i don't think so, but since you had it there, i moved those tests here as well in case there was something i was not seeing